.changeset/cool-wolves-serve.md

```diff
@@ -0,0 +1,6 @@
+---
+"@aws-amplify/ui-react-storage": patch
+"@aws-amplify/ui-react": patch
```

Comment:
It looks like all the changes are in ui-react-storage, do we need to bump ui-react too?

Reply:
+1, should only have to bump `ui-react-storage` here

Reply:
Fixed!